sql/driver/driver_test.go

```diff
@@ -223,7 +223,7 @@ func TestInsecure(t *testing.T) {
 	defer func() {
 		_ = db.Close()
 	}()
-	if _, err := db.Exec("CREATE DATABASE t"); err != nil {
+	if _, err := db.Exec(`SELECT 1`); err != nil {
```

Comment:
yeah. I really wanted DB.Ping(), but that doesn't necessarily open a connection. Any statement that makes it to the server will do (and they all do).